templates/bin/fastcheck

```diff
@@ -30,3 +30,10 @@ coffeelint -f .coffeelint.json app/javascripts/**/*.coffee
 
 # for javascript
 yarn eslint app/javascripts spec/javascripts
+
+# for diaeresis characters
+output=$(find . -name '*.yml' -exec grep -nH "̈" {} +); echo "$output"; [ -z "$output" ]
+if [ $? -ne 0 ]; then
```

Comment:
```suggestion
if find . -name '*.yml' -exec grep -nH "̈" {} + ; then
```

No point for the complicated dance with storing the output and checking if it is empty, if we just want to print the matches and error out if there are any.

Reply:
It was also pointed out by [shellcheck](https://www.shellcheck.net/) as a code smell:
```
In templates/bin/fastcheck line 36:
if [ $? -ne 0 ]; then
     ^-- SC2181 (style): Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.
```

Reply:
I think we then need to switch the check:

```
➜  find . -name '*.yml' -exec grep -nH "̈" {} +
./config/locales/pdf.de-CH.yml:15:         Wir freuen uns, Ihnen hiermit einen temporären Punktepass zu überreichen.
./config/locales/mailers.de-CH.yml:81:      intro_html: Vielen Dank für Ihr Interesse an der Gratis Manor Treuekarte. Um
➜  echo $?                                    
0
```

```
➜  find . -name '*.yml' -exec grep -nH "̈" {} +
➜  echo $?                                    
1
```

because it's 0 if it finds something

Reply:
Ah sorry yeah. Should have tested.

Reply:
```
if find . -name '*.yml' -exec grep -nH "<0308>" {} +; then echo '!!!found!!!'; fi
./config/locales/mailers.de-CH.yml:75:        enjoy_html: Wir wünschen Ihnen viel Spass beim Shopping und Punktesammeln.
./config/locales/mailers.de-CH.yml:81:      intro_html: Vielen Dank für Ihr Interesse an der Gratis Manor Treuekarte. Um
./config/locales/pdf.de-CH.yml:15:         Wir freuen uns, Ihnen hiermit einen temporären Punktepass zu überreichen.
./config/locales/pdf.de-CH.yml:16:         Mit dem Punktepass können Sie von Rabatten profitieren, zeitlich befristet Manor Punkte sammeln und einsetzen,
./config/locales/pdf.de-CH.yml:17:         jedoch nicht Ihre Einkäufe bezahlen.
./config/locales/pdf.de-CH.yml:19:         damit Sie jederzeit von allen Vorteilen profitieren können.\n\n
./config/locales/pdf.de-CH.yml:20:         Der Punktepass ist während 14 Tagen ab Ausstellungsdatum gültig und wird in allen Manor Filialen akzeptiert.
./config/locales/pdf.de-CH.yml:21:         Bewahren Sie ihn sorgfältig auf.\n\n
./config/locales/pdf.de-CH.yml:22:         Wir wünschen Ihnen viel Vergnügen beim Einkaufen in unseren Warenhäusern."
./config/locales/pdf.de-CH.yml:44:         Die Manor Sofortkarte ist während 14 Tagen ab Ausstellungsdatum gültig und wird in allen Manor Warenhäusern akzeptiert.
./config/locales/mailers.fr-CH.yml:75:        enjoy_html: Wir wünschen Ihnen viel Spass beim Shopping und Punktesammeln.
!!!found!!!
```